package.json

```diff
@@ -70,9 +72,5 @@
   },
   "svelte": "./dist/index.js",
   "types": "./dist/index.d.ts",
-  "type": "module",
-  "dependencies": {
-    "highlight.js": "^11.10.0",
```

Comment:
It's good to remove them, but I'm a bit surprised we don't have anything in `dependencies` now. We should check this once we deploy the docs.